akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -363,7 +372,7 @@ class Source[+Out, +Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[Sour
    * @param n must be positive, and `d` must be greater than 0 seconds, otherwise [[IllegalArgumentException]] is thrown.
    */
   def groupedWithin(n: Int, d: FiniteDuration): javadsl.Source[java.util.List[Out @uncheckedVariance], Mat] =
-    new Source(delegate.groupedWithin(n, d).map(_.asJava)) // FIXME optimize to one step
+    new Source(delegate.groupedWithin(n, d).map(_.asJava)) // TODO optimize to one step
```

Comment:
yeah, TODO is not as urgent as FIXME (a final release should not contain any FIXMEs)